src/graphql/Raffle/Query.ts

```diff
@@ -0,0 +1,44 @@
+import { User, Context} from '../../utils'
+
+/**
+ * return all user
+ * or specific user
+ * @param parent 
+ * @param args 
+ * @param ctx 
+ */
+function getUser(parent: object, args: User, ctx: Context): User {
+  const {prisma} = ctx
+  const {id} = args
+
+  if(!id)
+    return prisma.users.findMany()
+   
+  return prisma.users.findOne({
+    where:{
+      id: Number(id)
+    }
+  })
+}
+
+/**
+ * return all
+ * users winner
+ * @param parent 
+ * @param args 
+ * @param ctx 
+ */
+function getAllWinner(parent: object, args: User, ctx: Context): User {
+  const {prisma} = ctx
+    
+  return prisma.users.findMany({
+    where:{
+      winner: 1
+    }
+  })
+}
+
+
+export const Query = {
```

Comment:
It's better to just write out the function definition inside of the Query object.

Like so:
```suggestion
export const Query = {
  getUser: (parent, args, ctx): User => { ... }
  // ...
}
```